### DISCUSSION TITLE
Better Test Request Popup

### DISCUSSION THREAD

**Discussion by itizarsa** [scalar/scalar#1781] [General]
I came across `fern-api-docs`, which has a super popup for trying the requests. It would be nice if Scalar provided similar to that. 

Also, the response properties rendered like the request properties look great

https://petstore-openapi.docs.buildwithfern.com/api-reference/pet/add-pet

**Comment by marclave** [comment]
hi @itizarsa , appreciate your feedback!

we have had a test request since launch in october :) is there something different than ours that you would like to see?

https://sandbox.scalar.com/e/eaOkZ

<img width="1512" alt="image" src="https://github.com/scalar/scalar/assets/6176314/5c6a67b2-1f81-4779-bf06-be2875e5053a">
